index.xml

```diff
@@ -9935,7 +10088,8 @@ counter.value;                                           <span class='comment'>/
               The <a class='xattr' href='#SecureContext'>[SecureContext]</a> extended attribute
               <span class='rfc2119'>MUST NOT</span> be specified on both an interface
               member and the interface or partial interface definition the
-              interface member is declared on.
+              interface member is declared within, or on both a namespace member and the namespace
+              definition the namespace member is declared within.
```

Comment:
Needs to mention partial namespaces.

But also, this is maybe changing the semantics.  With the new wording, this IDL:

    [SecureContext] interface Foo {};
    partial interface Foo {
      [SecureContext] void func();
    };

is allowed, I think, whereas I _think_ the intent is that it not be allowed or something.  Though it's not clear that the old language prohibited this either; the "declared on" concept is pretty vague.  :(